include/glow/Base/Type.h

```diff
@@ -356,6 +356,24 @@ inline std::pair<dim_t, dim_t> flattenCdr(llvm::ArrayRef<dim_t> dims,
   return {first, rest};
 }
 
+/// Collapse a tensor shape into two sizes: the rest will be
+/// size of n without the axis dimension, and  first will be
+/// size of the axis dimension. For example, ([7, 3, 4, 2], 1) -> [56, 3]
+inline std::pair<dim_t, dim_t> flattenCdrV1(llvm::ArrayRef<dim_t> dims,
```

Comment:
nit: Maybe naming here could be improved? Could be used outside of loading this specific op.

Reply:
named it collapseTensor. Let me know if you have any suggestions here.